newIDE/app/src/LocalApp.js

```diff
@@ -88,7 +88,10 @@ export const create = (authentification: Authentification) => {
         eventsFunctionsExtensionWriter={LocalEventsFunctionsExtensionWriter}
         eventsFunctionsExtensionOpener={LocalEventsFunctionsExtensionOpener}
```

Comment:
Same in this file, if no actual code changes, prefer to cancel them in your PR to avoid losing review time.

Reply:
Oh! I will remember in future to check for previous unwanted changes whenever a new update is made.

Reply:
These unwanted changes might have been due to my code editor's formatting settings. Will fix it for future,